tests/release_tests/test_launch/launch-config.yml

```diff
@@ -0,0 +1,93 @@
+apiVersion: v1
```

Comment:
Same here, we should document required changes to manifests for testing, but use the existing manifests